spec/lib/danger/request_sources/github_spec.rb

```diff
@@ -401,6 +401,42 @@
         allow(@g).to receive(:submit_pull_request_status!)
       end
 
+      context "with Octokit v6" do
+        it "passes relative line number in the diff hunk to client.create_pull_request_comment" do
+          stub_const("Octokit::MAJOR", 6)
+
+          allow(@g.client).to receive(:pull_request_comments).with("artsy/eigen", "800").and_return([])
+
+          allow(@g.client).to receive(:create_pull_request_comment)
+
+          expect(@g.client).to receive(:delete_comment).with("artsy/eigen", inline_issue_id_1).and_return({})
+          expect(@g.client).to receive(:delete_comment).with("artsy/eigen", inline_issue_id_2).and_return({})
+          expect(@g.client).to receive(:delete_comment).with("artsy/eigen", main_issue_id).and_return({})
+
+          v = Danger::Violation.new("Sure thing", true, "Rakefile", 34)
```

Comment:
The following diff is used as test fixture.

https://github.com/danger/danger/pull/412/files#diff-ee98e028c59b193d58fde56ab4daf54d43c486ae674e63d50ddf300b07943e0fR34